packages/types/src/localization.ts

```diff
@@ -14,6 +14,7 @@ export type LocalizationValue = string;
 export type LocalizationResource = DeepPartial<_LocalizationResource>;
 
 type _LocalizationResource = {
+  locale: 'en-US' | 'fr-FR' | 'de-DE' | 'da-DK' | 'es-ES' | 'it-IT' | 'nl-NL' | 'nl-BE' | 'pt-BR' | 'ru-RU' | 'sv-SE';
```

Comment:
Should we allow all values here for any custom implementations without a localization preset from us?

Reply:
@anagstef @anagstef would you like to just append string to the type so  the auto complete stays there but we continue to support any other locale

Reply:
@panteliselef Sounds good for me!

Reply:
Sure, if that works. We just don't want any ts errors popping up if they use some other locale.

Reply:
Valid request, I will update the PR before merging